tests/integration/cli/repos_test.py

```diff
@@ -16,13 +16,20 @@ def setUp(self):
         super(TestReposCLI, self).setUp()
         busybox_single_app_bundle_fixture(num_bundles=1)
         self.run_command('bundles activate busyboxa')
+
         self.temp_repos_dir = mkdtemp()
+        self.fake_override_dir = mkdtemp()
+        self.fake_from_dir = mkdtemp()
+
+        os.chmod(self.temp_repos_dir, 0777)
+        os.chmod(self.fake_override_dir, 0777)
+        os.chmod(self.fake_from_dir, 0777)
```

Comment:
made me notice, should tearDown instead `rmtree(fake_override_dir)` rather than `fake_override_repo_location` which lives inside `fake_override_dir`? Alternatively, would it work to just `chmod(fake_override_repo_location)`?